presto-server-rpm/pom.xml

```diff
@@ -169,12 +205,96 @@
                                 <rule>
                                     <destination>/usr/lib/presto/lib</destination>
                                 </rule>
-
                             </rules>
                         </package>
                     </packages>
                 </configuration>
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>ci</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>0.31.0</version>
+                        <configuration>
+                            <logDate>none</logDate>
+                            <images>
+                                <image>
+                                    <name>prestodev/centos7-oj8</name>
+                                    <run>
+                                        <ports>
+                                            <!--suppress UnresolvedMavenProperty -->
+                                            <port>${presto.port}:8080</port>
+                                        </ports>
+                                        <volumes>
+                                            <bind>
+                                                <volume>${project.build.directory}:/target</volume>
+                                            </bind>
+                                        </volumes>
+                                        <cmd>
+                                            <exec>
+                                                <args>sh</args>
+                                                <args>-c</args>
+                                                <args>
+                                                    rpm -i /target/${project.build.finalName}.x86_64.rpm
+                                                    /etc/init.d/presto start
+                                                    tail ---disable-inotify -F /var/log/presto/server.log
+                                                </args>
+                                            </exec>
+                                        </cmd>
+                                        <wait>
+                                            <time>60000</time>
+                                            <log>.*SERVER STARTED.*</log>
```

Comment:
we could configure hive connector to verify it at least loads
(at startup time, it requires no external services, so it's easy)

Reply:
Done